### ISSUE TITLE
Java Spring for ServiceBus, azure-servicebus-jms-spring-boot-starter, fails to load unless a second undocumented app setting is in place.

### ISSUE THREAD

**Issue by NickDrouin** [MicrosoftDocs/azure-dev-docs#292]
[Enter feedback here]

Follow the walkthrough at: 
https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-service-bus

This will fail unless the value for:
spring.jms.servicebus.connection-string
is replicated into:
azure.servicebus.jms.connection-string

Specifically, it will throw when loading the app.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dbecafd0-c6db-9cb0-c5a4-328946446ec6
* Version Independent ID: 40cea238-c2de-fd1d-fef7-c1a850b54158
* Content: [How to use the Spring Boot Starter for Azure Service Bus JMS](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-service-bus)
* Content Source: [articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-service-bus.md](https://github.com/MicrosoftDocs/azure-dev-docs/blob/master/articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-service-bus.md)
* Service: **azure-java**
* GitHub Login: @seanli1988
* Microsoft Alias: **seal**

**Comment by KarlErickson** [comment]
@NickDrouin thanks for the feedback! We'll investigate and update the topic.

**Comment by scgbear** [comment]
<details>
<summary>Internal Tracking</summary>
devdivcsef [387675]
</details>

**Comment by NickDrouin** [comment]
Product bug has been root-caused, see:
https://github.com/Azure/azure-sdk-for-java/issues/15333

It was the failure message that was referring to an old/different setting name.

I think this doc bug can be closed.

Note that I got into this problem when I used double quotes around the connection string.  It might be useful to future readers of this walkthrough to have actual strings in place of the appsettings.properties examples; new readers may not know that these values should be unquoated.
